### ISSUE TITLE
Add selftest for sync and aux pins

### ISSUE THREAD

**Issue by electroniceel** [GlasgowEmbedded/Glasgow#152]
Currently there is no selftest for the sync and aux pins of revC. These should be added.

For a loop test they should be connected like on the test jig pcb: sync to A0, Aux A10 to A1 and Aux C9 to A2.

These tests also require the internal pullups on the ice40 which are not yet supported in nmigen.

**Comment by whitequark** [comment]
Sync to A0?

**Comment by electroniceel** [comment]
Yeah. When executing "pins-sync-loop" the user is required to connect the sync pin to port A pin 0.

I don't like looping Aux together as they could probably have a short between them. Connecting them to port A, which was tested independently by the other tests, seems to be a better idea to me.

**Comment by whitequark** [comment]
Makes sense
